pyproject.toml

```diff
@@ -9,7 +9,7 @@ dev = [
     "pylint>=3.0.0a7",
     "ruff>=0.2.2",
     # required for running node/npm commands for local development if node is not installed globally:
-    "nodejs-bin[cmd]>=18.4.0a4",
+    "nodejs-wheel>=20.12.2",
```

Comment:
oops, my previous comment was on the wrong dependencies section. this is the one that should be removed, and the one in `build-system.requires` should be added back.
```suggestion
```

sorry about that. i can take over from here if you want

Reply:
sure feel free!